src/compiler/commandLineParser.ts

```diff
@@ -216,6 +216,13 @@ namespace ts {
             isCommandLineOnly: true,
             description: Diagnostics.Print_the_final_configuration_instead_of_building
         },
+        {
+            name: "listFilesOnly",
+            type: "boolean",
+            category: Diagnostics.Command_line_Options,
+            isCommandLineOnly: true,
+            description: Diagnostics.Print_names_of_files_that_are_part_of_the_compilation_and_then_stop_processing

```

Comment:
We should also make this Advanced options instead.

Reply:
Ignorant question: what's the difference?

Reply:
I think we concluded that `Command_line_Options` is fine.